satpy/writers/__init__.py

```diff
@@ -224,6 +231,12 @@ def add_overlay(orig, area, coast_dir, color=(0, 0, 0), width=0.5, resolution=No
 
         LOG.debug("Automagically choose resolution %s", resolution)
 
+    if hasattr(orig, 'convert'):
```

Comment:
wouldn't a try/except be more pythonic ?

Reply:
I suppose so. I've never really liked try/except for cases like this where I'm saying "does this have this functionality". Regardless, my only other worry would be an error in convert that would get caught and make it seem like the user needs to update trollimage.

Reply:
fine, I don't feel strongly about this